google-cloud-tools-plugin/src/com/google/gct/idea/appengine/cloud/DoManagedVmDeployment.java

```diff
@@ -122,6 +122,7 @@ public void run() {
     commandLine.withWorkDirectory(stagingDirectory);
     consoleLogLn(loggingHandler, "Working directory set to: " + stagingDirectory.getAbsolutePath());
     commandLine.withParentEnvironmentType(ParentEnvironmentType.CONSOLE);
+    commandLine.getEnvironment().put("CLOUDSDK_METRICS_ENVIRONMENT", "gcloud-intellij");
```

Comment:
To keep these PR's separate, its probably best to accept this one when ready, and I can merge from master into #541 and include the refactoring in there.